build-tools/packages/bundle-size-tools/src/ADO/index.ts

```diff
@@ -3,15 +3,24 @@
  * Licensed under the MIT License.
  */
 
-export * from "./AdoArtifactFileProvider";
-export * from "./AdoSizeComparator";
-export * from "./Constants";
-export * from "./DefaultStatsProcessors";
-export * from "./FileSystemBundleFileProvider";
-export * from "./getAzureDevopsApi";
-export * from "./getBuildTagForCommit";
-export * from "./getBundleBuddyConfigMap";
-export * from "./getBundleFilePathsFromFolder";
-export * from "./getBundleSummaries";
-export * from "./getCommentForBundleDiff";
-export * from "./PrCommentsUtils";
+export {
+    getBundleBuddyConfigFileFromZip,
+    getBundlePathsFromZipObject,
+    getStatsFileFromZip,
+    getZipObjectFromArtifact,
+} from "./AdoArtifactFileProvider";
+export { ADOSizeComparator } from "./AdoSizeComparator";
+export { IADOConstants, totalSizeMetricName } from "./Constants";
+export { DefaultStatsProcessors } from "./DefaultStatsProcessors";
+export {
+    getBundleBuddyConfigFromFileSystem,
+    getBundlePathsFromFileSystem,
+    getStatsFileFromFileSystem,
+} from "./FileSystemBundleFileProvider";
+export { getAzureDevopsApi } from "./getAzureDevopsApi";
+export { getBuildTagForCommit } from "./getBuildTagForCommit";
+export { GetBundleBuddyConfigMapArgs, getBundleBuddyConfigMap } from "./getBundleBuddyConfigMap";
+export { BundleFileData, getBundleFilePathsFromFolder } from "./getBundleFilePathsFromFolder";
+export { GetBundleSummariesArgs, getBundleSummaries } from "./getBundleSummaries";
```

Comment:
I typically do case-insensitive sort, with shorter items coming before longer items.  So e.g.

a
Ab
aBc

It looks like there are some places in this PR that follow this type of sorting already (build-tools/packages/version-tools/src/index.ts) but others that don't (here).

Reply:
sorting looks good in the other PR; https://github.com/microsoft/FluidFramework/pull/12371/files